Podcast/Player.swift

```diff
@@ -41,10 +35,13 @@ class Player: NSObject {
         didSet {
             autoplay = true
```

Comment:
can we move this to its own well named function instead of setting this in the didSet